composer/loggers/mosaicml_logger.py

```diff
@@ -44,7 +50,25 @@ class MosaicMLLogger(LoggerDestination):
     """Log to the MosaicML platform.
 
     Logs metrics to the MosaicML platform. Logging only happens on rank 0 every ``log_interval``
-    seconds to avoid performance issues.
+    seconds to avoid performance issues. The following metrics are logged upon ``INIT``:
```

Comment:
```suggestion
    seconds to avoid performance issues. 
    
    Additionally, the following config metrics are logged upon ``INIT``:
```

Reply:
Is this in addition to the argument?

Reply:
nope, these are all of the metrics that we parse out of the `analytics_data` variable (the arg) 👍